### PR TITLE
Update appnexusBidAdapter.js

### PR DESCRIPTION
**Pull request by bllmo**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Added Beintoo as Appnexus alias

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by jsnellbaker** [comment]
Just to note - waiting on internal review/approval before merging this new alias.

**Comment by bllmo** [comment]
@jsnellbaker do you have any news about this new alias?

**Comment by jsnellbaker** [comment]
@bllmo I have followed up again internally.  Will hopefully have an answer soon.

**Comment by jsnellbaker** [comment]
@bllmo when you have the chance, can you please resolve the conflicts?  We should be good to merge at that point.

File: modules/appnexusBidAdapter.js [hunk 1]
```diff
@@ -44,8 +44,7 @@ const storage = getStorageManager(GVLID, BIDDER_CODE);
 
 export const spec = {
   code: BIDDER_CODE,
-  gvlid: GVLID,
```
Comment (jsnellbaker) [inline modules/appnexusBidAdapter.js:47]: Please add this variable back into the spec object.

**Comment by ddantuonobeintoo** [comment]
Hi jsnellbaker let we know if now can be merge the code. Thanks

**Pull request merged**